src/index.ts

```diff
@@ -95,6 +97,7 @@ module.exports = class ServerlessPlugin {
     if (config.enableTags) {
       this.serverless.cli.log("Adding service and environment tags to functions");
       this.addServiceAndEnvTags();
+      this.addPluginTag();
```

Comment:
I think we should add the plugin version tag regardless of `config.enableTags`

Reply:
Good call.